src/renderers/dom/shared/DOMProperty.js

```diff
@@ -151,6 +151,9 @@ var DOMPropertyInjection = {
 };
 var defaultValueCache = {};
 
+var ATTRIBUTE_NAME_START_CHAR = ':A-Z_a-z\\u00C0-\\u00D6\\u00D8-\\u00F6\\u00F8-\\u02FF\\u0370-\\u037D\\u037F-\\u1FFF\\u200C-\\u200D\\u2070-\\u218F\\u2C00-\\u2FEF\\u3001-\\uD7FF\\uF900-\\uFDCF\\uFDF0-\\uFFFD';
+var ATTRIBUTE_NAME_CHAR = ATTRIBUTE_NAME_START_CHAR + '\\-.0-9\\uB7\\u0300-\\u036F\\u203F-\\u2040';
```

Comment:
Make a private variable that is not exported called ATTRIBUTE_NAME_START_CHAR, and include that variable in the object that gets exported.  Then you can include the private variable in the sum.